src/components/FamilyManifestModal/FamilyManifestModal.js

```diff
@@ -187,16 +187,17 @@ export default compose(
                 });
                 return withFormik({
                   handleSubmit: async (value, { setSubmitting, setErrors }) => {
-                    fileManifestParticipantsAndFamily({
-                      sqon: downloadSqon,
-                      columns: columns,
-                    })().then(
-                      async profile => {
+                    fileManifestParticipantsAndFamily().then(
```

Comment:
fileManifestParticipantsAndFamily is curried, so needs the execution brackets here